spec/actions/inject_into_file_spec.rb

```diff
@@ -70,6 +70,22 @@ def file
       expect(File.read(file)).to eq("__start__\nREADME\nmore content\n__end__\n")
     end
 
+    it "does not attempt to change the file if it doesn't exist" do
+      invoker.inject_into_file "idontexist", :before => "something" do
+        "any content"
+      end rescue nil
```

Comment:
I thought it was better to have separate `expect`s for the different aspects of the change, but I've adjusted the test file according to your comments now. let me know if you had something different in mind.
Thanks.